CMakeLists.txt

```diff
@@ -23,18 +23,31 @@ IF(RPM)
 	SET(CPACK_RPM_MXS_ADAPTER_PACKAGE_NAME "mariadb-columnstore-maxscale-cdc-adapters")
 	SET(CPACK_RPM_MXS_ADAPTER_FILE_NAME "${CPACK_RPM_MXS_ADAPTER_PACKAGE_NAME}-${CPACK_RPM_PACKAGE_VERSION}-${CPACK_RPM_PACKAGE_RELEASE}-${ENGINE_ARCH}-${RPM}.rpm")
 
-	# rename packets manually if CMAKE < 3.6.3
-	if(${CMAKE_VERSION} VERSION_LESS "3.6.3") 
+	# rename packets manually if CMAKE < 3.6.3 (CentOS 7)
+	if(${CMAKE_VERSION} VERSION_LESS "3.6.3")
+		if(${CMAKE_VERSION} VERSION_GREATER "3.0.1")
+			cmake_policy(SET CMP0037 OLD) 
+		endif()
+
 		SET( CPACK_OUTPUT_CONFIG_FILE "${CMAKE_BINARY_DIR}/RPMPackageConfig.cmake" )
 		include(CPack)
 
 		ADD_CUSTOM_TARGET ( package
```

Comment:
Doesn't CMake automatically generate a `package` target? Why is a custom command needed?

Reply:
Yes, but I've to execute cpack manually to be able to add custom commands (rename) after cpack is finished. Cmake 3.6.3 includes functions to rename components, earlier versions don't. I used the target package to be compatible between the cmake versions, otherwise it would be make package in cmake 3.6.3 and make "something else" in OSes with cmake < 3.6.3.